chia/rpc/full_node_rpc_api.py

```diff
@@ -63,8 +63,8 @@ async def _state_changed(self, change: str) -> List[WsRpcMessage]:
         return []
 
     async def get_initial_freeze_period(self):
-        freeze_period = self.service.constants.INITIAL_FREEZE_PERIOD
-        return {"INITIAL_FREEZE_PERIOD": freeze_period}
+        freeze_period = self.service.constants.INITIAL_FREEZE_END_TIMESTAMP
+        return {"INITIAL_FREEZE_END_TIMESTAMP": freeze_period}
```

Comment:
@seeden needs to support this since warning is based on FREEZE_PERIOD

Reply:
It does't look like it's used in main at the moment